docs/marketplace/dev/architecture-overview/202108.0/marketplace-architecture-overview-application-composition.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Marketplace Application Composition
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace looks like in general. After reading this odc you will now: 
+
+- how the Spryker Marketplace is composed on application level
+
+
+## Composition of Applications
+
+The following diagram explains the application composition, it is internationally simplified, learn more details on the [Marketplace infrastructure system requirements here](/docs/marketplace/dev/setup/system-infrastructure-requirements.html).
+
+![Applications composition](https://confluence-connect.gliffy.net/embed/image/3a83f861-b25e-4ef5-aee7-e7da0b182cfa.png?utm_medium=live&utm_source=custom)
+
+### Glue
+Is a customer-facing REST API-based store-front application. This is the place where customer browses catalog and make orders.
```

Comment:
It's Storefront API, glue is just an API framework that serves Storefront API and Backend API.